src/state/state.js

```diff
@@ -188,14 +188,14 @@
          * @ignore
          */
         function _renderFrame(time) {
-            // update all game objects
-            me.game.update(time);
-            // render all game objects
-            me.game.draw();
             // schedule the next frame update
             if (_animFrameId !== -1) {
                 _animFrameId = window.requestAnimationFrame(_renderFrame);
             }
+            // update all game objects
+            me.game.update(time);
+            // render all game objects
+            me.game.draw();
```

Comment:
It's supposed to create a smoother animation than when you draw after making the frame request, because you're making the request sooner. I've seen quite a few articles mention this.

http://www.paulirish.com/2011/requestanimationframe-for-smart-animating/